models/product/src/product-variant/presets/boring-generic-milk-master-variant.ts

```diff
@@ -1,16 +1,12 @@
+import * as Attribute from '../../attribute';
 import ProductVariant from '../builder';
 import { TProductVariantBuilder } from '../types';
 
 const boringGenericMilkMasterVariant = (): TProductVariantBuilder =>
   ProductVariant()
     .key('boring-generic-milk-master-variant-key')
     .sku('boring-generic-milk-master-variant-sku')
-    .attributes([
-      {
-        name: 'cow-name',
-        value: 'unknown',
-      },
-    ])
+    .attributes([Attribute.random().name('cow-name').value('unknown')])
```

Comment:
This is a buildable field so we can't provide a value object but use the appropriate test model.